.github/workflows/azure-e2e-test.yml

```diff
@@ -261,6 +268,8 @@ jobs:
           --gallery-image-definition ${{ secrets.AZURE_PODVM_IMAGE_DEF_NAME }} \
           --gallery-image-version "${PODVM_IMAGE_VERSION}" || true
 
+    - name: Remove container registry
```

Comment:
Nit: it's cleaner to do this adding of names in a different commit.

Reply:
The fix is somewhat related, since we still want to remove the container registry, even if we don't want to remove the podvm image. before this maybe not labeled properly but not a problem.

Reply:
OK. fair.

Reply:
This does not delete the entire registry, just the prebuilt image.